### DISCUSSION TITLE
How to Run Sadtalker on an AMD GPU

### DISCUSSION THREAD

**Discussion by xyyyuuan** [OpenTalker/SadTalker#556] [Q&A]
How to Run Sadtalker on an AMD GPU，
如何使用amdgpu运行Sadtalker，希望大佬解答一下

**Comment by fraz0815** [comment]
See this [pull request](https://github.com/OpenTalker/SadTalker/pull/567) at least for new Navi 31 cards. 
You will need to  delete your venv folder I think to get the new rocm packages installed.
